Makefile

```diff
@@ -134,3 +146,12 @@ publish-package-test: package ## upload a package to the testpypi
 
 publish-package: package ## upload a package
 	twine upload dist/*
+
+publish-docker: docker-build ## push the docker images
+    docker push $(DOCKER_NAME):python3
+	docker push $(DOCKER_NAME):python3-$(VERSION)
+	docker push $(DOCKER_NAME):python3-onbuild-$(VERSION)
+	docker push $(DOCKER_NAME):python2
```

Comment:
Having these at different indentations is probably not going to work in a Makefile... the first one probably needs to be a tab.